### PR TITLE
Lesson 1 complited

### PR DESCRIPTION
**Pull request by AlexeyBrambalov**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,13 +1,23 @@
...
 
 const App = () => {
   return (
     <div className="App">
-      <header className="App-header">CYF Hotel</header>
+      <Heading />
+      <TouristInfoCards />
       <Bookings />
+      <Footer
+        list={[
```
Comment (40thieves) [inline src/App.js:15]: Is this the best name for this prop? Does it describe what is actually being passed?

File: src/components/Footer.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
+import React from "react";
+
+const Footer = props => {
+  return (
+    <ul className="bg-dark text-white-50 text-center list">
+      {props.list.map((item, index) => (
+        <li key={index}>{item}</li>
```
Comment (40thieves) [inline src/components/Footer.js:7]: Great 👍

File: src/components/TouristInfoCards.js [hunk 3]
```diff
@@ -0,0 +1,43 @@
+import React from "react";
+
+function TouristInfoCards() {
```
Comment (40thieves) [inline src/components/TouristInfoCards.js:3]: This component renders all 3 tourist info cards. But if I wanted to show just 1 of the cards in another place in the app, I'd have to copy this code to where I wanted it.

Instead if we made a component that only rendered 1 card, and we passed it some props for the image, url and name then we can re-use the component where-ever we want.

File: src/App.js (hunk 1 above)
Comment (AlexeyBrambalov) [reply src/App.js]: yes, I updated it

File: src/components/TouristInfoCards.js (hunk 3 above)
Comment (AlexeyBrambalov) [reply src/components/TouristInfoCards.js:4]: yes, I updated it as a new component TouristInfoCard

**Pull request closed without merging**